instrumentor/internal/webhook_env_injector/webhook_env_injector.go

```diff
@@ -139,3 +147,28 @@ func getContainerEnvVarPointer(containerEnv *[]corev1.EnvVar, envVarName string)
 	}
 	return nil
 }
+
+func InjectPythonNativeEnvVars(container *corev1.Container) {
+	container.Env = append(container.Env,
+		corev1.EnvVar{
+			Name: "NODE_IP",
+			ValueFrom: &corev1.EnvVarSource{
+				FieldRef: &corev1.ObjectFieldSelector{
+					FieldPath: "status.hostIP",
+				},
+			},
+		},
+		corev1.EnvVar{
+			Name:  commonconsts.OpampServerHostEnvName,
+			Value: fmt.Sprintf("$(NODE_IP):%d", commonconsts.OpAMPPort),
```

Comment:
WDYT about making this environment variable only host name, this way we don't need any formatting.